cmd/init/init.go

```diff
@@ -72,8 +72,8 @@ func GetBSFInitializers() (bsfv1.SearchServiceClient, *hcl2nix.FileHandlers, err
 }
 
 // CleanUp removes the bsf config if any error occurs in init process (ctrl+c or any init process stage)
-func cleanUp(){
-	configs := []string{"bsf", "bsf.hcl", "bsf.lock"}
+func cleanUp() {
+	configs := []string{"bsf", "bsf.hcl", "bsf.lock", ".envrc"}
```

Comment:
Let's not clean up `.envrc`. User might their own or modified it beyond bsf's context